CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ else()
 endif()
 
 add_subdirectory(app)
-
+set_target_properties(app PROPERTIES OUTPUT_NAME ${BIN_NAME})
```

Comment:
What is this used for?

Reply:
due the cross compilation, the build is basically running twice and to ensure that both binaries are kept, I added this to have different names for the different architectures